build/controllers/scalingo.js

```diff
@@ -0,0 +1,64 @@
+const dayjs = require('dayjs');
+const slack = require('../../build/controllers/slack');
+const {
+  Message,
+  Section,
+  Context,
+  Attachment,
+} = require('slack-block-builder');
+const config = require('../../config');
+
+function getSlackMessageAttachments(payload) {
+  const appName = payload.app_name;
+  const scalingoAppUrl = `${config.scalingo.oscFr1Url}/${appName}/`;
+  const appActivityUrl = `${scalingoAppUrl}activity`;
+  const appDeploymentUrl = `${scalingoAppUrl}deploy/${payload.type_data.deployment_uuid}`;
+  const deploymentStatus = payload.type_data.status;
+  const pusher = payload.type_data.pusher;
+  const gitRef = payload.type_data.git_ref;
+  const userName = payload.user?.username;
+  const eventDate = dayjs(payload.type_data.finished_at).format('MMM D');
+  const message = `[${appName}] App deployment error`;
```

Comment:
Si c'est plus clair?
```suggestion
  const { 
   status: deploymentStatus, 
   pusher, 
   get_ref: gitRef, 
   deployment_uuid: uuid, 
   finished_at: finishedAt 
 } = payload.type_data;
  const appName = payload.app_name;
  const scalingoAppUrl = `${config.scalingo.oscFr1Url}/${appName}/`;
  const appActivityUrl = `${scalingoAppUrl}activity`;
  const appDeploymentUrl = `${scalingoAppUrl}deploy/${uuid}`;
  const userName = payload.user?.username;
  const eventDate = dayjs(finishedAt).format('MMM D');
  const message = `[${appName}] App deployment error`;
```

Reply:
IMO je trouve ça plus dur à lire